### PR TITLE
Plot pane

### PR DESCRIPTION
**Pull request by davidanthoff**
Not perfect, and the UI needs some work, but I think this can be merged for now. Addresses #164.

It seems to work with Gadfly, PyPlot, VegaLite, Plots (with backends GR, PlotlyJS, some others don't work, but I think that might be more of a Plots.jl with julia 0.6 problem right now).

### CODE REVIEW

**Comment by ZacLN** [comment]
I'm getting a 'can't' connect: no such file' error, I can't get much more detail as the terminal closes instantly. I seem to remember a difference in pipes between linux/windows?

**Comment by ZacLN** [comment]
Scratch that, on linux the naming/positioning of the pipe seems to be identical to how it is on windows

File: scripts/terminalserver/terminalserver.jl [hunk 1]
```diff
@@ -1,5 +1,9 @@
 module _vscodeserver
 
+import Base: display, redisplay
+
+immutable InlineDisplay <: Display end
+
 pid = Base.ARGS[1]
 
 if is_windows()
```
Comment (ZacLN) [inline scripts/terminalserver/terminalserver.jl]: change to `if is_windows() || is_unix()` and drop the following if branch

**Comment by davidanthoff** [comment]
@ZacLN Can you try again? I think it now should all work, plus UI is all finished as well.

**Comment by ZacLN** [comment]
I'm just on a mobile 'til Monday I'm afraid so can't test things

**Comment by davidanthoff** [comment]
Ah, ok. Ok if I just merge it anyways, without a review? I tested it on Ubuntu and it all seems to work. If there is a bug we can still fix it later?

**Comment by ZacLN** [comment]
Yep for sure

**Comment by KristofferC** [comment]
Maybe tag a new alpha release? I want to try with  https://github.com/KristofferC/PGFPlotsX

**Comment by davidanthoff** [comment]
@KristofferC https://github.com/JuliaEditorSupport/julia-vscode/releases/tag/v0.6.0-alpha.3

**Pull request merged**